left_vector_length: 0.15
right_vector_length: 0.15
pattern_angle_between_lr: 0
distance_tolerance: 0.1
detect_angle_tolerance: 0.15
angle_threshold: 0.5
pattern_distance_lr: 0.407
laser_frame_id: back_laser_link
dock_frame: parallel_frame